backend/spindle/views.py

```diff
@@ -52,14 +51,24 @@ class SpindleResponse:
 
     def json_response(self) -> JsonResponse:
         # TODO: set HTTP error code when error is not None
+
+        # Convert display_type to displayType for frontend.
+        camelized = [
+            {
+                **{k: v for k, v in phrase.items() if k != "display_type"},
```

Comment:
to me this seems overly complicated (or very much typescript-flavored)
how about:

```python
for phrase in self.lexical_phrases:
   phrase["displayType"] = phrase.pop("display_type")
```

Reply:
Yes, this is much more readable 😅